skyportal/models/photometry.py

```diff
@@ -248,3 +263,720 @@ def snr(self):
+            )
+
+        self._data.to_netcdf(full_name)
+
+        self.filename = full_name
+
+    def delete_data(self):
+        """
+        Delete the underlying photometric data from disk
+        """
+
+        if os.path.exists(self.filename):
+            os.remove(self.filename)
+
+    read = (
+        accessible_by_groups_members
+        | accessible_by_streams_members
+        | accessible_by_owner
+    )
+    update = delete = accessible_by_owner
+
+    filename = sa.Column(
+        sa.String,
+        nullable=False,
+        index=True,
+        doc="Full path and filename, or URI to the netCDF file storing photometric data.",
+    )
+
+    mjd_first = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc='MJD of the first exposure of the series.',
+        index=True,
+    )
+
+    mjd_mid = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc='MJD of the middle of the observation series.',
+        index=True,
+    )
+
+    mjd_last = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc='MJD of the last exposure of the series.',
+        index=True,
+    )
+
+    mjd_last_detected = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc='MJD of the last exposure that was above threshold.',
+        index=True,
+    )
+
+    detected = sa.Column(
+        sa.Boolean,
+        nullable=False,
+        doc='True if any of the data points are above threshold.',
+        index=True,
+    )
+
+    series_identifier = sa.Column(
+        sa.String,
+        nullable=False,
+        doc='Unique identifier of the series of images out of which the photometry is generated.',
+        index=True,
+    )
+
+    series_obj_id = sa.Column(
+        sa.String,
+        nullable=False,
+        doc='Unique identifier of an object inside the series of images out of which the photometry is generated.',
+    )
+
+    channel_id = sa.Column(
+        sa.String, nullable=False, default='0', doc='Channel of the photometric series.'
+    )
+
+    filter = sa.Column(
+        allowed_bandpasses,
+        nullable=False,
+        doc='Filter with which the observation was taken.',
+    )
+
+    exp_time = sa.Column(
+        sa.Float, nullable=False, doc='Median exposure time of each frame, in seconds.'
+    )
+
+    frame_rate = sa.Column(
+        sa.Float,
+        nullable=False,
+        doc='Median frame rate (frequency) of exposures in Hz.',
+    )
+
+    num_exp = sa.Column(
+        sa.Integer, nullable=False, doc='Number of exposures in the series. '
+    )
+
+    time_stamp_alignment = sa.Column(
+        time_stamp_alignment_types,
+        nullable=False,
+        doc='When in each exposure is the mjd timestamp measured: start, middle, or end.',
+        default='middle',
+    )
+
+    ra_unc = sa.Column(sa.Float, doc="Uncertainty of ra position [arcsec]")
+    dec_unc = sa.Column(sa.Float, doc="Uncertainty of dec position [arcsec]")
```

Comment:
These are calculated right? if ra_unc, dec_unc is missing in the data then this will be None -- it should be nullable